src/Symfony/Component/PropertyAccess/Tests/StringUtilTest.php

```diff
@@ -67,6 +68,7 @@ public function singularifyProvider()
             array('emphases', array('emphas', 'emphase', 'emphasis')),
             array('faxes', 'fax'),
             array('feet', 'foot'),
+            array('feedback', 'feedback'),
```

Comment:
Exactly, I was just wondering if having the same word for a singular and a plural would not lead to issues when using such code. `getFeedback()` -> plural or singular? `$this->feedback` -> plural or singular?